samples/rtl/axpy_double_pump.py

```diff
@@ -0,0 +1,456 @@
+import dace
+from dace.transformation.dataflow import StreamingMemory
+from dace.transformation.interstate import FPGATransformState
+from dace.transformation.dataflow import TrivialMapElimination
```

Comment:
```suggestion
```

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/11984d3c2c997860d2124ef82133077ccd84e753